watchdog-config/extra-sql-prometheus.yml

```
@@ -1,5 +1,6 @@
 scrape_configs:
   - job_name: 'extra-sql'
-    scrape_interval: 12h
+    scrape_interval: 300s
+    scrape_timeout: 180s
```

While fine to keep at `300s` for testing, please don't commit this value!  `12h` is more than enough given the data!  Also `12h` will greatly reduce the load on the RDBMS server

```suggestion
    scrape_interval: 12h
    scrape_timeout: 180s
```

---

Updated